run-tests.sh

```diff
@@ -58,7 +61,8 @@ fi
 for arg in "$@"
 do
     case $arg in
-        --check-shellscript) check_script;;
+        --check-commitlint) check_commitlint "$@";;
```

Comment:
The passing of `$@` to the function breaks when multiple operation are requested, like this:
```
./run-tests.sh --check-pytest --check-commitlint
```

Maybe we can remove the for loops that iterates over all the arguments? I personally never pass multiple checks at the same time, so maybe also others do not do that?

Reply:
Yes, I'm doing the same as you, so I agree. Changed.